eng/MSBuild/ProjectStaging.props

```diff
@@ -0,0 +1,28 @@
+<Project>
+
+  <PropertyGroup>
+    <!-- Amend the description based on stage -->
+    <Description Condition="'$(Stage)' == 'dev'">Experimental package. $(Description)</Description>
+    <Description Condition="'$(Stage)' == 'obsolete'">Obsolete Package. $(Description)</Description>
+
+    <_IsStable Condition="('$(Stage)' != 'dev' and '$(Stage)' != 'preview') Or '$(MSBuildProjectName)' == 'Microsoft.AspNetCore.Testing'">true</_IsStable>
+
+    <!--
+        When DotNetFinalVersionKind is set to 'release' (only for the release branches),
+        the build will produce stable outputs for 'Shipping' packages.
+
+        In 8.0, we shipped Microsoft.AspNetCore.Testing package as stable even when it was in the dev stage, so we
+        keep it as stable for compatiblity.
+      -->
+    <DotNetFinalVersionKind Condition=" '$(configureDotNetFinalVersionKind)' == 'true' And '$(DotNetFinalVersionKind)' == '' And '$(_IsStable)' == 'true' ">release</DotNetFinalVersionKind>
+
+    <!-- Preview packages: do not use stable branding and do not warn about lack of [Experimental] -->
+    <NoWarn Condition="'$(Stage)' == 'dev' or '$(Stage)' == 'preview'">$(NoWarn);LA0003</NoWarn>
+    <!--
+        Makes it such that the package version won't be stabilized even when the rest of the repo is going stable.
+        https://github.com/dotnet/arcade/blob/main/Documentation/CorePackages/Versioning.md#package-version
+      -->
+    <SuppressFinalPackageVersion />
+    <SuppressFinalPackageVersion Condition=" '$(_IsStable)' != 'true' ">true</SuppressFinalPackageVersion>
```

Comment:
You shouldn't need to set this. By default when DotNetFinalVersionKind is not release then packages will not be marked as stable, so no need to set this explicitly unless the rest of the repo is going stable and you want to opt-out in a package. I'm pretty sure you can remove both of these lines (25-26) and the behavior should be the same.